plugin/tools.py

```diff
@@ -731,6 +778,58 @@ def get_clang_version_str(clang_binary):
             raise RuntimeError(
                 " Couldn't find clang version in clang version output.")
 
+    @classmethod
+    def get_clang_version_str_linux(cls, output_text):
+        """Get Clang version string from output of "clang_binary -v".
+
+        This is the linux-specific variant.
+
+        Args:
+            output_text (str): output of "clang_binary -v"
+
+        Returns:
+            str: clang version number
+
+        Raises: RuntimeError: There is an error while getting version. This is
+            too important to continue. If this fails the plugin will not work
+            at all.
+        """
+        # now we have the output, and can extract version from it
+        version_regex = re.compile("\d\.\d\.*\d*")
+        match = version_regex.search(output_text)
+        if match:
+            version_str = match.group()
+            return version_str
+        else:
+            raise RuntimeError(
+                " Couldn't find clang version in clang version output.")
+
+    @classmethod
+    def get_clang_version_str_windows(cls, output_text):
```

Comment:
Yeah, windows and linux seem to behave the same for now. I'll change this to call into the linux variant.